src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();
+    private static final ColorSupplier colorService = new ColorSupplier();
+    private final int COUNT = 4;
```

Comment:
1. Constant should be static
2. Better to have more precise naming
```suggestion
    private static final int FIGURES_COUNT = 4;
```

Reply:
Ok